docs/man/rpm-config.5.scd

```diff
@@ -100,15 +100,15 @@ opposed opposed to just package building) parts:
 	- *-1*: (or undefined) autodetect on platforms where supported,
 	  otherwise default to disabled
 
-*%\_netsharedpath* _PATH_
+*%\_netsharedpath* _PATHS_
```

Comment:
This is an interesting question - I deliberately used non-plural `PATH` throughout the man page (there are multiple such instances) for the association with `PATH` environment variable in shell because it's the same exact form: a colon separated list of paths (sometimes just directories, sometimes files, sometimes either). Whether that was a good idea or not, I don't know :smile: